### ISSUE TITLE
generate thumbnails from our contributors

### ISSUE THREAD

**Issue by hannesdatta** [tilburgsciencehub/website#651]
With a new batch of RAs, let's update this:

![image](https://user-images.githubusercontent.com/16557730/224756832-ba75f464-aa57-4ca7-a339-3653710c419b.png)

**Comment by shrabasteebanerjee** [comment]
Indeed - this is related to this open issue as well : https://github.com/tilburgsciencehub/website/issues/643

**Comment by hannesdatta** [comment]
I need an AI that tells me when I'm creating duplicate issues...

**Comment by lachlandeer** [comment]
is there a way that the thumbnails can be generated automatically from either: 

* a list of the last `x` unique contributors
* the current profile pics of the team, which will be developed in #643 

Then we won't need to revist this over time

**Comment by hannesdatta** [comment]
I would try this w/ a Python script and add it to our actions. @thierrylahaije, any thoughts?

**Comment by thierrylahaije** [comment]
@hannesdatta I started of with this issue. I added a development branch to the issue where the first draft is visible. I still need to add the flags, which will be coming in the next update. To make the flags work, we will need to add some parameter which contains the contributors' origin. This is a preview:

<img width="498" alt="image" src="https://github.com/tilburgsciencehub/website/assets/65568251/5f3e3bcc-900b-4fcc-b770-fb555cf233ea">
